addons/binding/org.openhab.binding.knx/src/main/java/org/openhab/binding/knx/internal/handler/LoadState.java

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright (c) 2010-2018 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.knx.internal.handler;
+
+/**
+ * Enumeration containing the load states
+ *
+ * @author Karel Goderis - Initial contribution
+ */
+public enum LoadState {
+    L0(0, "Unloaded"),
+    L1(1, "Loaded"),
+    L2(2, "Loading"),
+    L3(3, "Error"),
+    L4(4, "Unloading"),
+    L5(5, "Load Completing");
+
+    private int code;
+    private String name;
```

Comment:
@kgoderis I was searching for uses of this Enum to find out how the `name` property was used. The `name` property and `getName(..)` methods of this Enum are confusing because an Enum already has a `.name()` method. Maybe the property can be renamed to something else so there will be no confusion?